lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -1,8 +1,10 @@
+<%page expression_filter="h"/>
 <%inherit file="/main.html" />
 <%namespace name='static' file='/static_content.html'/>
 <%!
 from django.utils.translation import ugettext as _
 from django.core.urlresolvers import reverse
+from openedx.core.djangolib.markup import HTML, Text
```

Comment:
Don't import things you aren't using. I don't see any references to HTML or Text.